app/models/test_track/session.rb

```diff
@@ -64,9 +62,11 @@ def sign_up!(*args) # rubocop:disable Metrics/MethodLength
     end
```

Comment:
If we're going to 3.0 should we rip out this deprecated syntax support? Any other deprecation warnings like this?

Reply:
👍 I think that's the only one.

Reply:
(hopefully that gets us back under Rubocop's ABC/MethodLength thresholds too)